plugins/node/opentelemetry-instrumentation-pg/test/pg.test.ts

```diff
@@ -640,4 +640,33 @@ describe('pg', () => {
       });
     });
   });
+
+  describe('Instrumentation with requireParentSpan', () => {
```

Comment:
For completeness, I would move these tests to #client.connect block and replicate them in #client.query block (ofc with querying the client instead of just connecting).